class X

	def main is shared
		pass

	void foo  # .error. Try "def"
		print 'hi'
